### PR TITLE
Create 6b6 multiplication table project

### PR DESCRIPTION
**Pull request by kristjantoots**

### CODE REVIEW

**Comment by mrdks00** [comment]
appi aia

File: src/03-exercises/Exercises/6by6MultiplicationTable/Program.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+            for (int i = 0; i <= 6; i++)
+            {
+                Console.Write(i + "\t");
+
+                for (int j = 1; j <= 6; j++)
+                {
+                    if (i > 0)
+                        Console.Write(i * j + "\t");
```
Comment (jalonenrichard) [inline src/03-exercises/Exercises/6by6MultiplicationTable/Program.cs]: Code should be in Estonian and leave in commented code

File: src/03-exercises/Exercises/6by6MultiplicationTable/Program.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        static void Main(string[] args)
+        {
+            // print out 6by6 multiplication table using two nested for loops
+
+
+            for (int i = 0; i <= 6; i++)
+            {
+                Console.Write(i + "\t");
+
+                for (int j = 1; j <= 6; j++)
```
Comment (lunts) [inline src/03-exercises/Exercises/6by6MultiplicationTable/Program.cs]: Hea töö :)

File: src/03-exercises/Exercises/6by6MultiplicationTable/Program.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                {
+                    if (i > 0)
+                        Console.Write(i * j + "\t");
+
+                    else
+                        Console.Write(j + "\t");
+                }
+                Console.Write("\n");
```
Comment (Varavavaht) [inline src/03-exercises/Exercises/6by6MultiplicationTable/Program.cs]: "\n + \n"

**Pull request merged**